packages/terra-dialog/src/Dialog.scss

```diff
@@ -3,4 +3,19 @@
     height: 100%;
     width: 100%;
   }
+
+  .header { // May not be needed if action header is already themeable.
```

Comment:
In my opinion, I would still theme it. Terra-time-input had to apply individual styles to all of it's buttons and inputs so that it would all be aligned with each other.